src/Pos/Wallet/Web/ClientTypes.hs

```diff
@@ -202,27 +264,75 @@ instance Default CWalletMeta where
 -- | Client Wallet (CW)
 -- (Flow type: walletType)
 data CWallet = CWallet
-    { cwAddress :: !CAddress
-    , cwAmount  :: !CCoin
-    , cwMeta    :: !CWalletMeta
+    { cwAddress  :: !CWalletAddress
+    , cwMeta     :: !CWalletMeta
+    , cwAccounts :: ![CAccount]
     } deriving (Show, Generic, Typeable)
 
 -- | Query data for wallet creation
--- (wallet meta + backup phrase)
 data CWalletInit = CWalletInit
-    { cwBackupPhrase :: !BackupPhrase
-    , cwInitMeta     :: !CWalletMeta
+    { cwInitMeta   :: !CWalletMeta
+    , cwInitWSetId :: !(CAddress WS)
     } deriving (Show, Generic)
 
 -- | Query data for redeem
 data CWalletRedeem = CWalletRedeem
-    { crWalletId :: !CAddress
+    { crWalletId :: !CWalletAddress
     , crSeed     :: !Text -- TODO: newtype!
     } deriving (Show, Generic)
 
+-- | Meta data of 'CWalletSet'
+data CWalletSetMeta = CWalletSetMeta
+    { cwsName         :: !Text
+    , cwsBackupPhrase :: !BackupPhrase
+    } deriving (Show, Eq, Generic)
+
+instance Default BackupPhrase where
+    def = mkBackupPhrase12
+        [ "transfer"
+        , "uniform"
+        , "grunt"
+        , "excess"
+        , "six"
+        , "veteran"
+        , "vintage"
+        , "warm"
+        , "confirm"
+        , "vote"
+        , "nephew"
+        , "allow"
+        ]
+
+instance Default CWalletSetMeta where
+    def = CWalletSetMeta "Personal Wallet Set" def
```

Comment:
Is this such a good idea? 
I mean, anyone could, at any point, reference this, and the wallet could be reconstructed using the public mnemonic strings above, right?
Isn't it better if we remove these two `Default` instances?